test/python/Dockerfile

```diff
@@ -180,6 +180,27 @@ RUN set -ex; \
   cd h-pipenv; \
   pipenv lock;
 
+
+#------------------------------
+# test i: mysqlclient, psycopg2
+#------------------------------
+FROM build as testi
+
+ARG BUILDPACK_DEBUG
+
+RUN install-tool python 3.11.0
+RUN install-tool python 3.10.8
+
+RUN python3.11 -m pip install mysqlclient psycopg2
+RUN python3.10 -m pip install mysqlclient psycopg2
+
+RUN set -ex; \
+  mkdir i; \
+  cd i; \
+  python3.11 -m pip freeze > libs3.11.txt; \
+  python3.10 -m pip freeze > libs3.10.txt
+
+
```

Comment:
Please update an existing pipenv or poetry test case instead.

Reply:
Can you tell me where I can find the code for `install-pip`? Searching the GitHub @containerbase org didn't yield any implementation (bash alias, function, or anything else). I'm asking, because I need to make sure that I'm attempting to install mysqlclient and psycopg2 with two different Python versions. That's in order to ensure that changes in `src/usr/local/buildpack/tools/v2/python.sh` contain the correct paths to the header files.

Reply:
for this you should add a bash test

https://github.com/containerbase/base/blob/2c1d9e037024b15d73ed6665dda541c8638a16ce/test/bash/tools/python.bats#L76

here is the bin path in bash tests:
https://github.com/containerbase/base/blob/2c1d9e037024b15d73ed6665dda541c8638a16ce/test/bash/util.sh#L8

so probably simply grep and validate the shell wrapper there

`install-tool` shell sript starts here:
https://github.com/containerbase/base/blob/2c1d9e037024b15d73ed6665dda541c8638a16ce/src/usr/local/bin/install-tool#L1

You don't need to go through the full code.